src/components/Profile/components/VictoryPromise/components/GraphDisplay.tsx

```diff
@@ -18,60 +19,57 @@ const GraphDisplay = ({ scores }: Props) => {
    */
   const emptySliceValue = minimumScore * 0.6;
 
+  Chart.register(RadialLinearScale, ArcElement, Tooltip, Legend);
```

Comment:
This can be moved outside the component so that the plugins are only registered once.

Reply:
You can disable the Legend plugin and not have to specify in the options that it shouldn't display anything.
```suggestion
  Chart.register(RadialLinearScale, ArcElement, Tooltip);
```